common/common.scss

```diff
@@ -1169,3 +1169,16 @@ section.tag-info {
 .embedded-posts.top .row .topic-body {
   border: none;
 }
+
+// Welcome Topic Banner
+.welcome-topic-banner {
+  @include fb-background();
+  border: 1px solid $border-color;
```

Comment:
We probably want to move this to CSS custom properties, but can be as a separate PR.